meta/main.yml

```diff
@@ -10,6 +10,10 @@ dependencies:
   - role: debops.golang
     tags: [ 'depend::golang', 'depend::golang:gitlab', 'depend-of::gitlab', 'type::dependency:hard' ]
 
+  - role: debops.nodejs
+    tags: [ 'depend::nodejs', 'depend::nodejs:gitlab', 'depend-of::gitlab', 'type::dependency:hard' ]
+    nodejs__upstream: True
```

Comment:
I see. In that case I think that adding some kind of tracking the fact that an upstream repository is needed and enabled in the `debops.nodejs`, say via local facts, might be the best solution. This clashes a bit with the idea that roles install the packaged versions by default, but GitLab is moving forward and not waiting for distributions to catch up. Pros and cons, I guess.

What Im wary of is use of GitLab with other applications/services on the same host and how using different NodeJS version might impact it. I think that having more logic in `debops.nodejs` that can handle an upgrade to upstream version (in case the distributed packages were installed first) and role knowing that upstream is installed and preferred would go a long way with solving this. Then, other roles could request an upstream to be present and this issue would be solved.